app/db/models/fund.py

```diff
@@ -17,6 +18,22 @@
 BaseModel: DefaultMeta = db.Model
 
 
+@dataclass
+class Organisation(BaseModel):
+    organisation_id = Column(
```

Comment:
Better to add the schema diagram into the docs folder.

Reply:
good idea, ill do that 👍